tests/test_runners/test_dotnet.py

```diff
@@ -94,6 +94,84 @@ def test_subset(self):
         output = "FullyQualifiedName!=rocket_car_dotnet.ExampleTest.TestAdd&FullyQualifiedName!=rocket_car_dotnet.ExampleTest.TestDiv\n"  # noqa: E501
         self.assertEqual(result.output, output)
 
+    @responses.activate
```

Comment:
Is this change from the PR previously merged?
https://github.com/launchableinc/cli/pull/588

Reply:
I rebased with the main branch and fixed the diff.